admin/google_search_console/class-crawl-issue-bulk.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @package WPSEO\Admin|Google_Search_Console
+ */
+
+/**
+ * Class WPSEO_Crawl_Issue_Marker
```

Comment:
Doesn't match the actual classname. I prefer to remove these from my PHPStorm templates because this comment doesn't actually add information.